pkg/oauth/apiserver/apiserver.go

```diff
@@ -29,7 +44,12 @@ type ExtraConfig struct {
 	makeV1Storage sync.Once
 	v1Storage     map[string]rest.Storage
 	v1StorageErr  error
+
+	postStartHooks map[string]genericapiserver.PostStartHookFunc
```

Comment:
don't we have these in the generic apiserver already? Why here too?

Reply:
there's a kind of weird do-once logic in 
`func (c *completedConfig) V1RESTStorage()`
that I did not want to touch